docker/webserver/batch_user.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+
+HTPASSWD_FILE="/etc/nginx/htpasswd/external/batch_api.htpasswd"
+
+if ! test -f "$HTPASSWD_FILE"
+then
+  touch "$HTPASSWD_FILE"
+fi
+
+if [ "$1" = "add_update" ]; then
+  /usr/bin/htpasswd -B "$HTPASSWD_FILE" "$2"
```

Comment:
-B / Bcrypt doesn't work for nginx I think (md5 is the 'best' it can do), I also made the mistake but double checked it by checking the code https://github.com/internetstandards/Internet.nl/issues/